src/connector/src/pulsar/source/reader.rs

```diff
@@ -78,6 +186,49 @@ impl SplitReader for PulsarSplitReader {
     where
         Self: Sized,
     {
-        todo!()
+        let (tx, rx) = mpsc::unbounded_channel();
+        let rx = UnboundedReceiverStream::from(rx);
+        let splits = try_match_expand!(_state, ConnectorStateV2::Splits).map_err(|e| anyhow!(e))?;
```

Comment:
suggest `_state` => `state` & `_props` => `props`

Reply:
I'll fix that